numba_dpex/core/parfors/parfor_lowerer.py

```diff
@@ -19,10 +19,11 @@
     ReductionKernelVariables,
 )
 from numba_dpex.core.utils.kernel_launcher import KernelLaunchIRBuilder
+from numba_dpex.dpctl_iface import libsyclinterface_bindings as sycl
```

Comment:
`from numba_dpex.core.datamodel.models import dpex_data_model_manager as dpex_dmm` is now unused please remove as part of the PR.

Reply:
And `_KernelArgs`

Reply:
yes